core/src/main/java/org/infinispan/reactive/publisher/impl/ClusterPublisherManagerImpl.java

```diff
@@ -861,6 +920,13 @@ public <R> SegmentPublisherSupplier<R> entryPublisher(IntSet segments, Set<K> ke
                                              deliveryGuarantee, batchSize, transformer);
    }
 
+   @Override
+   public CompletionStage<Long> sizePublisher(IntSet segments, Set<K> exclude, InvocationContext ctx, long flags) {
+      long addedFlags = EnumUtil.mergeBitSets(flags, FlagBitSets.CACHE_MODE_LOCAL);
+      return reduction(false, segments, null, ctx, addedFlags, DeliveryGuarantee.EXACTLY_ONCE,
+            sizeComposedType(), s -> PublisherReducers.count().apply(s), s -> PublisherReducers.add().apply(s));
```

Comment:
IIUC the transformer is not used in this case. I would suggest just setting it to null to confirm or even throwing an exception.